ironfish-cli/package.json

```diff
@@ -68,9 +68,7 @@
     "@oclif/plugin-warn-if-update-available": "3.1.8",
     "@types/keccak": "3.0.4",
     "@types/tar": "6.1.1",
-    "@zondax/ledger-ironfish": "0.1.2",
-    "@zondax/ledger-ironfish-dkg": "npm:@zondax/ledger-ironfish@0.4.0",
-    "@zondax/ledger-js": "^1.0.1",
```

Comment:
Bring this back

Reply:
We added this dependency because we needed access to `ResponseError`. Managing two versions of `@zondax/ledger-ironfish` caused problems with resolving to the right version, but we needed our own dependency for `ResponseError`

Reply:
I think that's fine, but you cannot use an import without referencing it in your package because if the library you depend on stops including it your code will stop compiling. You are making the code base depend on that library by importing it, therefore you need to add it to the package to indicate the dependency.

Reply:
A good way to see this in action is to comment out `import { ResponseError } from '@zondax/ledger-js'` in ledger.ts, then press CMD + . to trigger the autofix. Without this being in the package.json, it will not attempt to auto-import anything. If this is in the package.json, it correctly suggests to add the import.

Reply:
@mat-if tried this, it is not in this package.json . They haven't exposed it in `"@zondax/ledger-ironfish"`. I asked zondax to do it, and they said they should release it today